### PR TITLE
Alter the Apartment data model

### LINKED ISSUE #56: Update Apartment model
- change the connection with the User model to one-to-one.
- change all the variables names to fit the group coding standards.
- add an image filed

### PR DESCRIPTION
**Pull request by tamirho**
- Alter the owner filed to be a one-to-one connection with the User model.
- Change all the variables names to fit the group coding standards.
- Add a text filed to store image URL in DB.

Fix #56 

**Note:** Migrations files were not included since we still have a PR in that subject waiting for a response.

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -95,17 +95,17 @@ def __str__(self):
 
 
 class Apartment(models.Model):
-    owner = models.ForeignKey(AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='owner_apartments')
-    datePosted = models.DateTimeField(default=timezone.now, blank=True)
+    owner = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE)
+    date_posted = models.DateTimeField(default=timezone.now, blank=True)
     city = models.ForeignKey(City, on_delete=models.RESTRICT, related_name='city_apartments')
     address = models.TextField()
-    rentPricePerMonth = models.IntegerField()
```
Comment (AmitAharoni) [inline users/models.py:102]: I think `Apartment` and` Seeker` shared fields names should be matched.
rentPricePerMonth - rent.
content - about.

We better select one and updated the needed model.
What do u think?
@tamirho

**Comment by AmitAharoni** [comment]
PR looks good.
Please tag the mentors as reviewers.

**Review by Micha546 — COMMENTED** [review]
Looks good, can we use the image_url field to hold more than one url?
if not maybe we should add more url fields for more pictures? 🥇

**Comment by nadavsu** [comment]
> Looks good, can we use the image_url field to hold more than one url?

I think we should start with one and see how that goes.

@tamirho , LGTM!

**Review by nadavsu — APPROVED** [review]

File: users/models.py (hunk 1 above)
Comment (tamirho) [reply users/models.py:102]: Done

**Review by DanielMy777 — COMMENTED** [review]
Looks great!
Don't forget to migrate, and add the migration file as a dependency to the test file :)

File: users/models.py [hunk 2]
```diff
@@ -95,17 +95,17 @@ def __str__(self):
...
-    numOfRooms = models.IntegerField()
-    startDate = models.DateField()
-    content = models.TextField(blank=True)
-    isRelevant = models.BooleanField(default=True, blank=True)
-    # img = models.ImageField(default='default.jpg', upload_to='aprt_pics')
+    rent = models.IntegerField()
+    num_of_roomates = models.IntegerField()
+    num_of_rooms = models.IntegerField()
+    start_date = models.DateField()
+    about = models.TextField(blank=True)
```
Comment (Omeramsc) [inline users/models.py:106]: Either add a `max_length=300` or remove the length limit on the `seeker` model.

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Please make sure the issue is linked. 
I see that you tried, but I think you should remove the colon in order for it to work.

File: users/models.py (hunk 2 above)
Comment (tamirho) [reply users/models.py:106]: Ok, I deleted the length limit on the `seeker` model.

**Review by simzacks — APPROVED** [review]
LGTM

**Comment by OrAsaf** [comment]
LGTM

**Pull request merged**